packages/react/src/Switch/README.md

```diff
@@ -0,0 +1,5 @@
+# React Switch Component
+
+The React `<Switch />` component is a wrapper around the `<input type="checkbox" />`. It can be used as a standalone component or as a form control element. When used as a standalone component provide a `aria-labeledby` prop to keep it accessible. When used as a formcontrol, make sure to add the associated `<Label>` component.
+
```

Comment:
I think the docs on how developers should use this component should be in the story, no?

Reply:
What I meant is: how developers should use this component should be part of the story, so we can add it there and remove this part of the readme:

> It can be used as a standalone component or as a form control element. When used as a standalone component provide an `aria-label` or `aria-labelledby` prop to keep it accessible. When used as a formcontrol, make sure to add the associated `<FormLabel>` component.

Reply:
There is actually an a empty label in the switch. How should we solve this? I don't think two labels is desirable.

Reply:
If we want to use this both as a standalone component and a form control element, we do need 2 labels in the second case, right?

Standalone: the empty label in Switch is used for the visual representation of the switch, and if you click it it should change the checkbox. It's possible to do this in other ways, but I think this is one of the most concise ways. 

Form control: if you want to use the Switch with a visible text label, you can add a `FormLabel` to it. This means the input has 2 labels, but that is valid html. We should check how screenreaders handle this though, [this used to cause issues on Safari + VoiceOver](https://scottaohara.github.io/tests/html-inputs/double-label.html)